build.gradle.kts

```diff
@@ -20,6 +20,7 @@ configurations {
 dependencies {
   annotationProcessor("org.springframework.boot:spring-boot-configuration-processor")
 
+  implementation("uk.gov.justice.service.hmpps:hmpps-sqs-spring-boot-starter:1.3.0")
```

Comment:
@dhirajshettymoj we're not doing SQS? It's S3. The guide is for SQS/SNS but it's not what we're needing to do.

Reply:
arr hence no secrets for SQS

Reply:
thanks @psoleckimoj I have updated the PR.